sbncode/LArG4/CMakeLists.txt

```diff
@@ -31,6 +31,20 @@ cet_build_plugin(MergeSimSourcesSBN art::EDProducer
   BASENAME_ONLY
 )
 
+cet_build_plugin(SimpleMerge art::EDProducer
```

Comment:
This is the first time I see `art::EDProducer` here... I have learned something.

Reply:
yeah I just copied another producer that worked upon building !